envs/conda_build_config.yaml

```diff
@@ -240,8 +240,8 @@ scipy:
 sentencepiece:
   - 0.1.96
 setuptools:
-  - 50.3.*   # [not s390x]
-  - 51.*     # [s390x]
+  - 58.*     # [not s390x]
```

Comment:
I have intentionally not removed the selectors here for now. We can remove these after verification.